package.json

```diff
@@ -11,7 +11,7 @@
     "node": ">=v6.10.2"
   },
   "scripts": {
-    "docs": "node lib/bin/docs.js",
+    "docs": "npm run build; node lib/bin/docs.js",
```

Comment:
Can you please change from `;` to `&&` ? I don't think we'd want/need the docs script to run if building failed, unless I'm missing something?